example/build.gradle.kts

```diff
@@ -0,0 +1,105 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    compileSdk = libs.versions.exampleCompileSdk.get().toInt()
+    buildToolsVersion = libs.versions.buildTools.get()
+
+    defaultConfig {
+        applicationId = "com.example.afterpay"
+        minSdk = libs.versions.minSdk.get().toInt()
+        targetSdk = libs.versions.exampleCompileSdk.get().toInt()
+
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArguments["clearPackageData"] = "true"
+    }
+
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro",
+            )
+        }
+
+        create("staging") {
+            initWith(getByName("debug"))
+        }
+    }
+
+    sourceSets {
+        val main by getting
+        main.java.srcDirs("src/main/kotlin")
+        val androidTest by getting
+        androidTest.java.srcDirs("src/androidTest/kotlin")
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = libs.versions.java.get()
+    }
+
+    buildFeatures {
+        viewBinding = true
+    }
+
+    testOptions {
+        execution = "ANDROIDX_TEST_ORCHESTRATOR"
+    }
+
+    namespace = "com.example.afterpay"
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().all {
+    kotlinOptions.freeCompilerArgs += listOf("-Xallow-result-return-type")
+}
+
+dependencies {
+    implementation(project(":afterpay"))
```

Comment:
nit: add `enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")` to `settings.gradle.kts` so this becomes `projects.afterpay`?